WcaOnRails/config/locales/en.yml

```diff
@@ -383,6 +384,7 @@ en:
         competitor_limit_reason: "What is the reason for the limit on competitors? Please fill this out in English!"
         venueAddress: ""
         championship_type: ""
+        registration_requirements: "Please indicate here any specificities about the registration process, as needed per <a href=\"https://www.worldcubeassociation.org/regulations/guidelines.html#2c1+\" target=\"_blank\">Guideline 2c1+</a>. After the competition is announced, this field will <b>not</b> be editable anymore, and Board's approval will be needed to modify it.<br/>For example indicate here if competitors need to have a clean payment history with a specific entity, or if they will have to answer a confirmation email in case the maximum number of registrations is reached."
```

Comment:
Does this render correctly? Shouldn't this be called `registration_requirements_html`?

It also might be nice to mention here that the registration requirements will also show up on the register page, which raises the question of what language this should be written in. I guess you should tell people to fill this out both in English, and in the language your competitors speak?

Reply:
It does render correctly actually.
The examples [here](http://guides.rubyonrails.org/i18n.html#using-safe-html-translations) only include explicit use through `t`, and here it's a hint for a field, maybe that's why.